src/similar-npm-packages.ts

```diff
@@ -567,6 +597,16 @@ export const categories: Categories = {
     similar: ["raphael", "snapsvg", "two.js"],
   },
 
+  "template-engine": {
+    name: "Template engines",
+    tags: [
+      { tag: "template", weight: Weight.MAX },
+      { tag: "partial", weight: Weight.HIGH },
+      { tag: "html", weight: Weight.NORMAL },
```

Comment:
Let's keep the category for HTML templating languages separate.

You might also want to add "engine" as a keyword.